environment_setup/requirements.txt

```diff
@@ -1,8 +1,7 @@
-pytest==4.3.0
+pytest>=5.3
 requests>=2.22
-azureml>=0.2
 azureml-sdk>=1.0
```

Comment:
We need azureml. Otherwise we are getting exceptions like "ModuleNotFoundError: No module named 'numpy'"

Reply:
This package is extremely dated and no longer maintained.  Could we please at least try to follow the trail of adding `numpy` or other packages which come up?